### PR TITLE
fix: inverted index file not found

### PR DESCRIPTION
**Pull request by longjiquan**
issue: https://github.com/milvus-io/milvus/issues/29654

### CODE REVIEW

File: internal/core/thirdparty/tantivy/tantivy-wrapper.h [hunk 1]
```diff
@@ -182,7 +190,8 @@ struct TantivyIndexWrapper {
     finish() {
         if (!finished_) {
             tantivy_finish_index(writer_);
-            reader_ = tantivy_create_reader_for_index(writer_);
+            writer_ = nullptr;
+            reader_ = tantivy_load_index(path_.c_str());
```
Comment (czs007) [inline internal/core/thirdparty/tantivy/tantivy-wrapper.h:194]: finished_ = True

**Comment by samhuang-z** [comment]
rerun ut

**Pull request merged**